addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -448,7 +450,8 @@ public void parseAndUpdate(String request, String payLoad, String result) {
                     }
                     case TESLA_CHARGE_STATE: {
                         chargeState = gson.fromJson(result, ChargeState.class);
-                        ChannelUID theChannelUID = new ChannelUID(getThing().getUID(), "charge");
+                        ChannelUID theChannelUID = new ChannelUID(getThing().getThingTypeUID(), getThing().getUID(),
```

Comment:
Sorry, you did this change because I [mistakenly put the deprecated annotation on the wrong constructor](https://github.com/eclipse/smarthome/commit/d99ad13ca813b9d31f84fca0598a8225f04240ad#commitcomment-16735837)...

There is actually a much nicer way to do the updateState, see [here](https://github.com/eclipse/smarthome/pull/1224/files#diff-2ba4d1bdb2c89e2c8b66b342a88ed23eL314).